dex/overlays/osc/osc-cl1/configmaps/files/config.yaml

```diff
@@ -53,6 +53,12 @@ staticClients:
       - http://grafana-opf-monitoring.apps.odh-cl1.apps.os-climate.org/login/generic_oauth
     secretEnv: GRAFANA_SECRET
 
+  - id: openmetadata-airflow
+    name: openmetadata-airflow
+    redirectURIs:
+      - http://airflow-openmetadata.apps.odh-cl2.apps.os-climate.org/oauth-authorized/dex
```

Comment:
@MightyNerdEric I know that change has been merged but I wonder if it should be targeting `odh-cl1` instead of `odh-cl2` as callback URL ?